playground/frontend/playground_components/lib/src/controllers/example_loaders/standard_example_loader.dart

```diff
@@ -50,17 +50,29 @@ class StandardExampleLoader extends ExampleLoader {
 
   Future<void> _load() async {
     try {
-      final example = await exampleCache.getPrecompiledObject(
+      final exampleBase = await exampleCache.getPrecompiledObject(
         descriptor.path,
         descriptor.sdk,
       );
 
       _completer.complete(
-        exampleCache.loadExampleInfo(example),
+        await exampleCache.loadExampleInfo(exampleBase),
       );
 
       // ignore: avoid_catches_without_on_clauses
-    } catch (ex, trace) {
+    } catch (_) {
```

Comment:
This way we lose details of the first exception when loading a precompiled object and only report the exception of loading a snippet. So far we do not use those details anyway and only report "Error loading example" but I do not like losing it.

Options are:
- Synthesize an exception to contain multiple exceptions, throw it here.
- Add both exceptions to `ToastNotifier` here. I do not like this because it mixes responsibilities.

Or we can just add a TODO to preserve both exceptions later.

Reply:
From `completeError` doc: